src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -175,8 +175,8 @@ private static int doTypeSwitch(Object target, int startIndex, Object[] labels)
      * arguments are an array of case labels which must be non-null and of type
      * {@code String} (representing enum constants) or {@code Class}.
      * <p>
-     * The type of the returned {@code CallSite}'s method handle will have
-     * a return type of {@code int}.   It has two parameters: the first argument
+     * The returned {@code CallSite}'s method handle will have
+     * a return type of {@code int} and accepts two parameters: the first argument
```

Comment:
By writing the javadoc text above, it came to mind that, perhaps, some validation is in order for the static arguments. For instance:

* the enum type of a Class parameter doesn't match that of the BSM
* the static arguments contain more than one Class parameter (I think even if they are all of the same "correct" type, that's bogus, right?)